Sources/Client/WebSocket/WebSocket.swift

```diff
@@ -41,6 +41,8 @@ final class WebSocket {
     
     private let Timer: Timer.Type
     
+    private var typingEventTimeoutTimerControls: [User: TimerControl] = [:]
```

Comment:
WebSocket doesn't support multiple users, and we can only send users on behalf of logged in user. Why do we need a dictionary?

Reply:
It's for other users. Here we handle typing events from other users.